docs/source/api/models/submodels/transport_efficiency/index.rst

```diff
@@ -5,4 +5,11 @@ transport_efficiency
   :maxdepth: 1
 
   base_transport_efficiency
-  bruggeman_transport_efficiency
+  bruggeman
+  cation_exchange_membrane
```

Comment:
Could you rename the heading from `transport_efficiency` to `Transport Efficiency`? This has caused errors previously with duplicate headings in a file and its toctree